docs/DeveloperGuide.md

```diff
@@ -300,38 +300,34 @@ It will also include settings to create a custom look for the application.
+| `* * *`  | student living in dorm                    | edit a contact’s room number                                         | I can stay updated if my dorm mate changes room                                                 |
+| `* * *`  | student living in dorm                    | view all contacts                                                    | I can keep track of how to find my dorm mates if I need their help AND remember their birthdays |
+| `* *`    | student living in dorm                    | add when my friends are free                                         | I can remember when my friends are available                                                    |
+| `* *`    | student living in dorm                    | view which of my friends are free at a certain time                  | I know who I can ask to meet for leisure or work                                                |
+| `* *`    | student living in dorm                    | update when my friends are free                                      | I ensure that the free times stored are accurate                                                |
+| `* *`    | new user of this application              | view allowed commands when the application launches                  | I am aware of what functions I can use in the application                                       |
+| `* *`    | user exploring this application           | get autocomplete when typing commands                                | I can quickly give my commands                                                                  |
+| `* *`    | student living in dorm with many contacts | search a contact by name                                             | I can quickly find details of my dorm mates                                                     |
+| `* *`    | student living in dorm with many contacts | filter contacts by dorm room number                                  | I can find where are my dorm mates                                                              |
+| `* *`    | student living in dorm with many contacts | search a contact by birthday                                         | I know whose birthday is in which month                                                         |
+| `* *`    | student living in dorm                    | add a profile picture for each contact                               | I can recognise and identify the contact person                                                 |
+| `* *`    | student living in dorm                    | add telegram handle for each contact                                 | I can contact them on telegram as an alternative to their phone number                          |
+| `*`      | student living in dorm                    | sync room updated room numbers across multiple users (decentralised) | I can just update my room number and not worry about other user’s room numbers                  |
```

Comment:
I'm not sure what does it mean by "sync room updated room numbers across multiple users".

Reply:
removed